tokio/src/io/util/mem.rs

```diff
@@ -47,15 +47,33 @@ use std::{
 #[derive(Debug)]
 #[cfg_attr(docsrs, doc(cfg(feature = "io-util")))]
 pub struct DuplexStream {
-    read: Arc<Mutex<Pipe>>,
-    write: Arc<Mutex<Pipe>>,
+    read: Arc<Mutex<SimplexStream>>,
+    write: Arc<Mutex<SimplexStream>>,
 }
 
-/// A unidirectional IO over a piece of memory.
+/// A unidirectional pipe to read and write bytes in memory. It can be constructed
+/// by [`simplex`] function which will create a pair of reader and writer or by
+/// calling [`SimplexStream::new_unsplit`] that will create a handle for both
+/// reading and writing.
```

Comment:
Everything on the first markdown line is visible in the module docs.

![image](https://github.com/user-attachments/assets/d84bd5c5-08cb-4a70-8cab-78d6bd57fcdd)

Please move everything after the first period into a new paragraph.
```suggestion
/// A unidirectional pipe to read and write bytes in memory.
///
/// It can be constructed by [`simplex`] function which will create a pair of
/// reader and writer or by calling [`SimplexStream::new_unsplit`] that will
/// create a handle for both reading and writing.
```

Reply:
Ops, should be fixed.